twill-core/src/main/java/org/apache/twill/internal/Configs.java

```diff
@@ -31,6 +31,11 @@
      */
     public static final String JAVA_RESERVED_MEMORY_MB = "twill.java.reserved.memory.mb";
 
+    /**
+     * Set this to false to disable the secure store updates done by default.
+     */
+    public static final String ENABLE_DEFAULT_SECURE_STORE_UPDATE = "twill.enable.default.secure.store.update";
```

Comment:
I think the common convention is to call it like `twill.secure.store.update.location.enabled` (from widest to narrowest scope, see hadoop-default.xml). Explicitly calling out `location` because that is what get updated by default right now and potentially there can be other secure store gets updated by default.